deploy/998_initialize_contracts.ts

```diff
@@ -0,0 +1,27 @@
+import { DeployFunction } from 'hardhat-deploy/types';
+import { HardhatRuntimeEnvironment } from 'hardhat/types';
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const { deployer } = await hre.getNamedAccounts();
+
+  const { redeployedContracts, redeployedAssetStorageContracts, contractsForReinitialization } = hre.helpers;
+
+  if (!['hardhat', 'otp_testnet', 'otp_mainnet'].includes(hre.network.name)) {
+    const hubControllerAddress = hre.helpers.contractDeployments.contracts['HubController'].evmAddress;
+    const HubController = await hre.ethers.getContractAt('HubController', hubControllerAddress, deployer);
+
+    const setAndReinitializeContractsTx = await HubController.setAndReinitializeContracts(
+      redeployedContracts,
```

Comment:
As we are redeploying contetntAsset contract, to make dkg.js work without interruptions, we need to add to this list oldContentAsset contract address to be set in hub under different name. I guess we can do it here for this release and after that open PR to remove that step